### ISSUE TITLE
do not pass -std=gnu11 to C++

### ISSUE THREAD

**Issue by dimpase** [Macaulay2/M2#964]
This happens in mathic:
```
g++ -std=gnu11 -g3 -O2 -Wno-deprecated -I/home/dimpase/tmp/M2/M2/include -I/home/dimpase/tmp/M2/M2/include -I/home/dimpase/tmp/M2/M2/usr-host/include -I/home/dimpase/tmp/M2/M2/usr-host/include/cdd  -I/home/dimpase/tmp/M2/M2/submodules/memtailor/src  -I/home/dimpase/tmp/M2/M2/submodules/mathic/src  -I/home/dimpase/tmp/M2/M2/submodules/mathicgb/src  -I/home/dimpase/tmp/M2/M2/submodules/fflas_ffpack/src -isystem /usr/include/libxml2     -I/usr/src/gtest  -DNDEBUG -Wall -I /home/dimpase/tmp/M2/M2/usr-host/include -Wno-uninitialized -Wno-unused-parameter -std=c++11 -O2 -MM -c src/stdinc.cpp > bin/release/stdinc.d.tmp
cc1plus: warning: command line option ‘-std=gnu11’ is valid for C/ObjC but not for C++
```
Potentially it is a problem, as `-std=` is just ignored here. The correct value is `-std=gnu++11`.

**Comment by DanGrayson** [comment]
Could you look at the output from "make" and tell me the last directory "entered"?

**Comment by dimpase** [comment]
oops, sorry, it's in frobby (libraries/frobby/build/frobby_v0.9.0).
In `Makefile` you see 
```
cflags = $(CFLAGS) $(CPPFLAGS) -Wall -I $(GMP_INC_DIR)  \
         -Wno-uninitialized -Wno-unused-parameter -std=c++11
```
and then `$(CXX) ${cflags} ...`

**Comment by DanGrayson** [comment]
Frobby hasn't been updated since 2012, and if this issue, which I reported to Bjarke in 2017, if it ever prevents compilation, it will be trivial to patch the library to work around it.  (Bjarke now works for google.)

So I propose leaving it the way it is for now.

**Comment by dimpase** [comment]
well, it makes C++ use the default std value, which might be not compatible
with what you set for the rest of M2, and produce either linking errors or
interesting runtime errors.

**Comment by DanGrayson** [comment]
Good point, reopening.

**Comment by DanGrayson** [comment]
fixed in commit 16e262beed4c8b4a1c6a707acfeb5d50728eb57c
